ocaml/idl/datamodel_types.ml

```diff
@@ -179,6 +180,12 @@ let release_order_full = [{
       version_major = 2;
       version_minor = 7;
       branding   = "XenServer 7.2";
+    }; {
+      code_name     = Some rel_inverness;
```

Comment:
@kc284 How about a comment like this:
`(* For future XenServer versions, use the version number of the latest release, and "Unreleased" for the branding. *)`